resources/views/emails/new_reply.blade.php

```diff
@@ -6,6 +6,10 @@
 {{ $reply->excerpt(200) }}
 @endcomponent
 
+@if($thread->isAuthoredBy($user))
```

Comment:
Would you be able to write a test to make sure this is only sent to the author of the thread?